sql/types/number.go

```diff
@@ -61,10 +61,19 @@ var (
 
 	// decimal that represents the max value an uint64 can hold
 	dec_uint64_max = decimal.NewFromInt(math.MaxInt64).Mul(decimal.NewFromInt(2).Add(decimal.NewFromInt(1)))
+	dec_uint32_max = decimal.NewFromInt(math.MaxInt32).Mul(decimal.NewFromInt(2).Add(decimal.NewFromInt(1)))
+	dec_uint16_max = decimal.NewFromInt(math.MaxInt16).Mul(decimal.NewFromInt(2).Add(decimal.NewFromInt(1)))
```

Comment:
A bunch of these are unused (and most of the ones below here look plain wrong)

Reply:
deleted unused